.azure/pipelines/jobs/default-build.yml

```diff
@@ -218,21 +216,38 @@ jobs:
             Token: $(dn-bot-dnceng-artifact-feeds-rw)
 
     - ${{ if ne(parameters.steps, '')}}:
-      - ${{ parameters.steps }}
+      - ${{ each step in parameters.steps }}:
```

Comment:
Add a comment explaining what you're doing here

Reply:
Sorry, missed your comment. I'll do this in a quick follow-up PR.